docs/release-notes/8.18.asciidoc

```diff
@@ -1,6 +1,24 @@
 [[release-notes-header-8.18.0]]
 == 8.18
 
+[discrete]
+[[release-notes-8.18.3]]
+=== 8.18.3
+
+[discrete]
+[[enhancements-8.18.3]]
+==== Enhancements
+* Adds `dns` event collection for macOS for {elastic-defend} ({kibana-pull}223566[#223566]).
+* Adds pricing information about Elastic Managed LLM in AI Assistant and Attack Discovery tours and callouts ({kibana-pull}221566[#221566]).
+
+[discrete]
+[[bug-fixes-8.18.3]]
+==== Fixes
+* Fixes a bug where OSS models didn't work when streaming was ON ({kibana-pull}224129[#224129]).
+* Fixes a bug where cell actions didn't work when opening a timeline from specific rules ({kibana-pull}223302[#223302]).
+* Fixes an issue where the entity risk score feature stopped persisting risk score documents ({kibana-pull}221937[#221937]).
+* Fixes rule upgrade Threat Match mapping components overflowing on small screens ({kibana-pull}218628[#218628]).
```

Comment:
possible revision 
cc: @dplumlee 

```suggestion
* Fixes overflow issues with the threat match mapping component when you viewed the rule upgrade flyout on a smaller screen ({kibana-pull}218628[#218628]).
```

Reply:
This is probably a better way to phrase it for release notes